stacks/lemp/service-config/nginx.conf

```diff
@@ -0,0 +1,82 @@
+worker_processes 4;
```

Comment:
I'd rather use `/run` if `/run` were a tmpfs, the way it is guaranteed to be on modern distros, but it sounded like Kenton was unenthusiastic about having a second tmpfs mount in the sandbox.  So this is my compromise.